integration/client/src/main/java/com/cerner/beadledom/integration/client/IntegrationClientFeature.java

```diff
@@ -0,0 +1,18 @@
+package com.cerner.beadledom.integration.client;
+
+import com.google.inject.BindingAnnotation;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A default client feature annotation.
+ *
+ * @author Nick Behrens
+ */
+@BindingAnnotation
+@Target({ElementType.FIELD, ElementType.PARAMETER, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface IntegrationClientFeature {
```

Comment:
Can this be package-private?

Reply:
yup: https://github.com/cerner/beadledom/pull/149/commits/934f3528830366a99897f34926cc5ff9398dd98a